commands/utils/params.go

```diff
@@ -180,36 +185,40 @@ func (jp *JFrogPlatform) setDefaultsIfNeeded() (err error) {
 	return
 }
 
-type GitClientInfo struct {
+type Git struct {
 	GitProvider vcsutils.VcsProvider
 	vcsclient.VcsInfo
-	RepoName  string   `yaml:"repoName,omitempty"`
-	Branches  []string `yaml:"branches,omitempty"`
-	RepoOwner string
-}
-
-type Git struct {
-	GitClientInfo            `yaml:",inline"`
-	BranchNameTemplate       string `yaml:"branchNameTemplate,omitempty"`
-	CommitMessageTemplate    string `yaml:"commitMessageTemplate,omitempty"`
-	PullRequestTitleTemplate string `yaml:"pullRequestTitleTemplate,omitempty"`
-	EmailAuthor              string `yaml:"emailAuthor,omitempty"`
-	AggregateFixes           bool   `yaml:"aggregateFixes,omitempty"`
+	RepoOwner                string
+	RepoName                 string   `yaml:"repoName,omitempty"`
+	Branches                 []string `yaml:"branches,omitempty"`
+	BranchNameTemplate       string   `yaml:"branchNameTemplate,omitempty"`
+	CommitMessageTemplate    string   `yaml:"commitMessageTemplate,omitempty"`
+	PullRequestTitleTemplate string   `yaml:"pullRequestTitleTemplate,omitempty"`
+	EmailAuthor              string   `yaml:"emailAuthor,omitempty"`
+	AggregateFixes           bool     `yaml:"aggregateFixes,omitempty"`
 	PullRequestID            int
 }
 
-func (g *Git) setDefaultsIfNeeded(gitClientInfo *GitClientInfo) (err error) {
-	g.RepoOwner = gitClientInfo.RepoOwner
-	g.GitProvider = gitClientInfo.GitProvider
-	g.VcsInfo = gitClientInfo.VcsInfo
+func (g *Git) setDefaultsIfNeeded(gitParamsFromEnv *Git) (err error) {
+	g.RepoOwner = gitParamsFromEnv.RepoOwner
+	g.GitProvider = gitParamsFromEnv.GitProvider
+	g.VcsInfo = gitParamsFromEnv.VcsInfo
 	if g.RepoName == "" {
-		if gitClientInfo.RepoName == "" {
+		if gitParamsFromEnv.RepoName == "" {
 			return fmt.Errorf("repository name is missing. please set the repository name in your %s file or as the %s environment variable", FrogbotConfigFile, GitRepoEnv)
 		}
-		g.RepoName = gitClientInfo.RepoName
+		g.RepoName = gitParamsFromEnv.RepoName
 	}
 	if len(g.Branches) == 0 {
-		g.Branches = append(g.Branches, gitClientInfo.Branches...)
+		if len(gitParamsFromEnv.Branches) == 0 {
```

Comment:
g.Branches = gitParamsFromEnv.Branches
if len(g.Branches) == 0 {
var checkoutBrunch string
			if checkoutBrunch, err = GetCheckoutBranchFromDotGit(); err != nil {
				return
			}
g.Branches = []string{checkoutBrunch}
}

Reply:
@sverdlov93, the len check is a validation that the branches weren't received from the frogbot-config.yml file